magic-mock/public/index.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html>
+    <head>
+        <title>Magic Version 999.99</title>
+    </head>
+    <body>
+        <h1>Welcome to Magic Version 999.99</h1>
```

Comment:
Don't use magic ... use Client portal or public portal something like that ... We want to get rid of magic ...